static/_redirects

```diff
@@ -445,25 +444,24 @@
 /horizontal-pod-autoscaler/     /docs/tasks/run-application/horizontal-pod-autoscale/ 301
 /kubernetes/     /docs/ 301
 /kubernetes-bootcamp/*     /docs/tutorials/kubernetes-basics/ 301
-/kubernetes/swagger-spec/     https://github.com/kubernetes/kubernetes/tree/master/api/swagger-spec/ 301
+# /kubernetes/swagger-spec/     https://github.com/kubernetes/kubernetes/tree/master/api/swagger-spec/ 301
 /kubernetes/third_party/swagger-ui/     /docs/reference/ 301
 /latest/docs/     /docs/home/ 301
 /media/     /docs/community     301
 /news/     /docs/community     301
 /resource-quota/     /docs/concepts/policy/resource-quotas/ 301
 /serviceaccount/token/     /docs/tasks/configure-pod-container/configure-service-account/ 301
-/swagger-spec/*     https://github.com/kubernetes/kubernetes/tree/master/api/swagger-spec/ 301
+# /swagger-spec/*     https://github.com/kubernetes/kubernetes/tree/master/api/swagger-spec/ 301
 /third_party/swagger-ui/*     /docs/reference/ 301
-/v1.1/docs/admin/networking.html     /docs/concepts/cluster-administration/networking/     301
+# /v1.1/docs/admin/networking.html     /docs/concepts/cluster-administration/networking/     301
```

Comment:
If you're happy to remove this line entirely, feel free.

Reply:
I will remove the commented lines if reviewers agree this makes sense.